qa/common/common.sh

```diff
@@ -8,10 +8,15 @@ source $BASEDIR/common/json.sh
 source $BASEDIR/common/rbd.sh
 source $BASEDIR/common/rgw.sh
 
-SALT_MASTER=$(cat /srv/pillar/ceph/master_minion.sls | \
-             sed 's/.*master_minion:[[:blank:]]*\(\w\+\)[[:blank:]]*/\1/' | \
-             grep -v '^$')
-
+MASTER_MINION_SLS=/srv/pillar/ceph/master_minion.sls
```

Comment:
We want the check to fail when DeepSea is not properly installed. Someone might have a `/srv/pillar/ceph` directory without a working DeepSea installation.